tensorflow_probability/python/bijectors/ordered.py

```diff
@@ -30,13 +30,12 @@
 
 
 class Ordered(bijector.Bijector):
-  """Maps a vector of increasing elements to an unconstrained vector.
+  """Deprecated. Use bijectors.Invert(bijectors.Ascending()) instead.
```

Comment:
Should the class raise (well, warn with) a DeprecationWarning? Though I note no-where else in the tfp code is that made use of.

Reply:
I think the best thing to do here is to use `tensorflow.python.util.deprecation` see e.g. https://github.com/tensorflow/probability/blob/master/tensorflow_probability/python/bijectors/affine_scalar.py#L55-L59